Create SECURITY.md (#1884)
trixi-framework/Trixi.jl: SECURITY.md

# Security Policy

The Trixi.jl development team takes security issues seriously. We appreciate
all efforts to responsibly disclose any security issues and will make every
effort to acknowledge contributions.

## Supported Versions

> The current stable release versions following SemVer are supported with
> security updates.

Reviewer: ```suggestion
The current stable release following [semantic versioning](https://semver.org/)
is supported with security updates.
```
?

Author: I added a link to the Julia interpretation of SemVer instead.

## Reporting a Vulnerability

To report a security issue, please use the GitHub Security Advisory 
["Report a Vulnerability"](https://github.com/trixi-framework/Trixi.jl/security/advisories/new)
tab.

We will send a response indicating the next steps in handling your report.
After the initial reply to your report, we will keep you informed of the
progress towards a fix and full announcement, and may ask for additional
information or guidance.

> Report security bugs in third-party modules to the person or team maintaining
> the module.

Reviewer: ```suggestion
Please report security bugs in third-party modules directly to the person
or team maintaining the module.
```
